final/js/tracking.js

```diff
@@ -96,25 +89,25 @@ Array.prototype.remove = function(object) {
       break;
     }
   }
-}
+};
 
 function randomTarget() {
+  var big = 800
```

Comment:
Change the random lines so they are very long (reach till the end of the screen)